README.md

```diff
@@ -15,7 +15,7 @@ build environments.
 
 ## Usage
 
-Put `[lein-protoc "0.5.0"]` into the `:plugins` vector of your project.clj.
+Put `[lein-protoc "0.1.0-SNAPSHOT"]` into the `:plugins` vector of your project.clj.
```

Comment:
needs the group:
```suggestion
Put `[com.circleci/lein-protoc "0.1.0-SNAPSHOT"]` into the `:plugins` vector of your project.clj.
```

i think this should be versioned as `"0.6.0"`:
* as it's got all the stuff from the real lein protoc thru `0.5`. could be confusing to have a visually downgraded version
* projects shouldn't have snapshots long term, only for experimentation. this is fine for deps-plus which is typically added to the user profile, but if i was to add this to one of our libraries, i'd want a stable version for reproducibility